.github/workflows/workflow-get-latest-deployed-versions-from-github.yml

```
@@ -0,0 +1,51 @@
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Get Latest Deployed Infra Version
+        id: get-infra-version
+        run: echo "version=$(gh variable get INFRA_VERSION_${{ inputs.environment }} --repo ${{ github.repository }} --json value --jq .value || echo '')" >> $GITHUB_OUTPUT
+
+      - name: Get Infra SHA
+        id: get-infra-sha
+        if: steps.get-infra-version.outputs.version != ''
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:2:20: Double quote to prevent globbing and word splitting

(shellcheck)